web/src/app/shared/services/http-client-interceptor/http-client-interceptor.service.ts

```diff
@@ -29,7 +29,7 @@ export class HttpClientInterceptor implements HttpInterceptor {
       } else if (error instanceof HttpErrorResponse && (error.status === 404) && error.url.indexOf('environment-information.json')) {
         // ignore this error
       } else {
-        this.errorsService.triggerError(NOTIFICATION_TYPES.GENERAL_ERR, error);
+        // this.errorsService.triggerError(NOTIFICATION_TYPES.GENERAL_ERR, error);
```

Comment:
Do we really want to ignore all errors from the interceptor?

Reply:
This could be confusing for users, but nearly all expectable errors should be covered on a more specific level.
The alternative would be to either check if there already is an alert (which is difficult because this general error is always covered before the specific one) or we check if the received error is one that we expect elsewhere (i personally could not think of an elegant and scalable way to do this) or last option which would be huge rework: put all http error handling in this function and show the coresponding error for every case (lots of work without real benefit or an elegant abd scalable solution)